presto-main/src/main/java/com/facebook/presto/operator/OperatorExecutionContext.java

```diff
@@ -27,16 +27,19 @@
     private final Lifespan lifespan;
     private final int stageId;
     private final int taskId;
+    private final boolean lifespanCommitRequired;
 
     @JsonCreator
     public OperatorExecutionContext(
             @JsonProperty("lifespan") Lifespan lifespan,
             @JsonProperty("stageId") int stageId,
-            @JsonProperty("taskId") int taskId)
+            @JsonProperty("taskId") int taskId,
+            @JsonProperty("lifespanCommitRequired") boolean lifespanCommitRequired)
```

Comment:
I would move this change to the `Add operator execution info for TableWriterOperator and DeleteOperator `